docs/src/proposals/secp256k1_instruction.md

```diff
@@ -0,0 +1,75 @@
+---
+title: secp256k1 builtin instruction
+---
+
+## Problem
+
+Need to efficiently perform multiple secp256k1 pubkey recovery operations (ecrecover) in a transaction.
```

Comment:
The problem described here is that your solution to the actual problem is missing.  What is the actual problem? I'll guess: "Serially computing N secp256k1 public key recovery operations (ecrecover) unnecessarily increases latency to O(N), where O(1) is possible if computed in parallel."

Reply:
computing in O(1) vs. O(N) is just a more specific version of saying compute efficiently.

Reply:
"Latency" being the operative word. This proposal assumes latency is the blocker. BPF instruction limit might be another, but don't need to parallelize to solve that one.